Rohit/PostgresAssignment/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "api_myschool",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/app/server.js",
```

Comment:
why do you have js files in src folder ? should we use dist as common practice ...

Reply:
resolved